app/templates/view_supplier_users.html

```diff
@@ -42,7 +42,7 @@
               'Last login',
               'Pwd changed',
               'Locked',
-              summary.hidden_field_heading("") if current_user.has_role('admin') else "Status"
+              summary.hidden_field_heading("") if current_user.has_any_role('admin', 'admin-ccs-category') else summary.hidden_field_heading("Change status")
```

Comment:
I think this will show a column like:

| Change status  |
| -------------- |
| active  |
| deactivated  |
| active  |

For users that _cannot_ change the status

Reply:
The header should be hidden except for screen readers - but yes, it's the wrong way round! Good spot (not sure how/if it worked beforehand...)